pandas/_libs/interval.pyx

```diff
@@ -101,7 +96,7 @@ cdef class IntervalMixin:
         Returns
         -------
         bool
-            True if the Interval is not closed on the left-side.
+            True if the Interval is closed on the left-side.
```

Comment:
This is technically correct with reverting, but the sentence is wrong. I fixed it in a cleanup pr. Should we avoid reverting this specific change or do a follow up?

Reply:
Feel free to push directly to this PR with those cleanups. Looks like [c9d4d3d](https://github.com/pandas-dev/pandas/pull/48116/commits/c9d4d3d474b70744cf25d36b0dbeaf356f828eaa) should successfully revert those PR in the OP, so we can include the cleanups here too.

Reply:
Done. I'll open a follow up about cleaning up the typing again. Made this consistent but probably not possible to avoid reverting too

Reply:
Okay sounds good to follow up with the typing then.